app/views/partials/repo_owning_teams.scala.html

```diff
@@ -16,20 +16,35 @@
 
 @import uk.gov.hmrc.cataloguefrontend.connector.GitRepository
 
+@import views.partials.RepoOwningTeamsHelper
 @(repo: GitRepository)
+@hasTooManyTeams=@{
```

Comment:
new lines round the `@(repo: GitRepository)` would help see that this is the constructor